site/zenodo_rdm/redirector.py

```diff
@@ -165,6 +174,25 @@ def record_export_view():
     return target
 
 
+def legacy_record_export_view():
+    """Implements redirector view function for legacy values of record export.
+
+    The following routes are redirected as follows:
+        -  /records/<pid_value>/export/<legacy_export_format> -> GET /record/<pid_value>/export/<export_format>
+
+    :return: url for the view 'invenio_app_rdm_records.record_export'
+    :rtype: str
+    """
+
+    args = request.view_args
+    export_format = request.path.split("/")[-1]
+    legacy_value = ZENODO_RECORD_EXPORTERS_LEGACY.get(export_format)
```

Comment:
out of curiosity, what would happen with invalid formats? Does the new endpoint handle it the same way as the previous?

Reply:
If the format is invalid, it just goes to 404. But in this case, I have specified this function only for the valid ones (for the reason explained above).